### PR TITLE
Laying the foundation for Bookstore Web App.

### PR DESCRIPTION
**Pull request by mailsg**
Hi,

This PR is raised to demonstrate laying foundation for a React Web App with initial components and routes with the use of `React Router` and `React Components`. Below mentioned are the implementations covered in the PR :
- Setup a new React application using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app)
- Create a directory for the reusable components : `/components`.
- Installed [React Router V6](https://reactrouter.com/en/main/start/tutorial#setup)
- Setup router inside of `<App>`
- Add 2 routes in the application, each should render a page component: 
  - `/`
  - `/categories`
- Created components inside of the `/components` directory: 

     1. Contains individual book state, as well as a button to delete the book
     2. Contains a list to renders individual books
     3. Contains a form to create new books
     4. Contains a navigation to link to the 2 routes created
     5. Replaced the input category with author.

Kindly review the PR and provide feedback on the same.

Thanks.

### CODE REVIEW

File: src/components/Books.jsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                <h2>{book.title}</h2>
+              </div>
+              <p>{book.author}</p>
+              <div className="updatebook">
+                <button type="button" id="comment">Comment</button>
+                <div className="left-section-1" />
+                <button type="button" id="remove">Remove</button>
+                <div className="left-section-1" />
+                <button type="button" id="edit">Edit</button>
+              </div>
```
Comment (victorokoroji) [inline src/components/Books.jsx]: - [ ] Good job so far 👍🏽 . I think you should create a separate component called `<Book />` which will be used to render a single book by passing props to it here in your `<Books />` component. This way your component will be reusable and clean.

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @mailsg, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the Git Flow ✅ 
- Project is properly initialized ✅ 
- The PR title and description are professional and descriptive ✅ 
- The README is well documented ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.jsx (hunk 1 above)
Comment (mailsg) [reply src/components/Books.jsx]: Done

**Review by TadesseDev — APPROVED** [review]
### Hi @mailsg 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅
## Optional suggestions

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**